pkg/regressionallowances/types.go

```diff
@@ -128,3 +147,26 @@ func addIntentionalRegression(release release, in IntentionalRegression) error {
 
 	return nil
 }
+
```

Comment:
Groundwork for [TRT-1448](https://issues.redhat.com/browse/TRT-1448) adds support for json array of intentional regressions